examples/symlink-write.js

```diff
@@ -1,25 +1,28 @@
 var fstream = require('../fstream.js')
 var notOpen = false
 
-fstream
-  .Writer({
-    path: 'path/to/symlink',
-    linkpath: './file',
-    isSymbolicLink: true,
-    mode: '0755' // octal strings supported
-  })
-  .on('close', function () {
-    notOpen = true
-    var fs = require('fs')
-    var s = fs.lstatSync('path/to/symlink')
-    var isSym = s.isSymbolicLink()
-    console.log((isSym ? '' : 'not ') + 'ok 1 should be symlink')
-    var t = fs.readlinkSync('path/to/symlink')
-    var isTarget = t === './file'
-    console.log((isTarget ? '' : 'not ') + 'ok 2 should link to ./file')
-  })
-  .end()
+// No symlinks on Windows
```

Comment:
Whilst Windows does have a selection of things that are like symlinks, none of them are quite the same, or node doesn't handle them properly, or they need permissions that are not available to regular users with the default installation options (and rarely in enterprise setups). They only work on NTFS, and often confuse the heck out of many tools which should know better. Apologies if my war wounds are flaring up, but I've concluded that they are more trouble than they are worth. However, if the security policy is changed, and node were fixed (possibly done already), true NTFS symbolic links are pretty close to unix symlinks.